crates/storage-query-datafusion/src/partition_state/schema.rs

```diff
@@ -53,8 +53,8 @@ define_table!(
         /// Replay status
         replay_status: DataType::Utf8,
 
-        /// Last persisted log LSN
-        persisted_log_lsn: DataType::UInt64,
+        /// Durable log LSN
+        durable_log_lsn: DataType::UInt64,
```

Comment:
This is a breaking change. This will change the sql schema

Reply:
Maybe check with @igalshilman what is the best approach to update this. Do we keep both names and mark the old name as deprecated, and keep it in code for couple of versions before removing ?

Reply:
I should have been more careful with this; I knew I am renaming the column in the view but switched it anyway. You're right that we should keep this as-is - reverted!

PS. I believe the versioning mechanism is purely about query data encoding, rather than the schema itself. At some point we might want to think about how we version the schema too, but I don't want to open that right now.